code/game/objects/items/devices/scanners.dm

```diff
@@ -401,3 +402,56 @@ FORENSIC SCANNER
 		printing.name = scan_name
 		printing.info = "Chemicals found: [dat]"
 		user.put_in_hands(printing)
+
+/obj/item/device/black_market_scanner
+	name = "suspicious device"
+	desc = "This is... seemingly a makeshift combination between an autopsy scanner, an ancient t-ray scanner, and some sort of robotic clamp, but you can see a lightbulb inside it. What the hell is this?"
+	icon_state = "mendoza_scanner"
+	item_state = "analyzer"
+	w_class = SIZE_SMALL
+	flags_atom = FPRINT|CONDUCT
+	flags_equip_slot = SLOT_WAIST
+	throwforce = 5
+	throw_speed = SPEED_VERY_FAST
+	throw_range = 20
+	matter = list("metal" = 60,"glass" = 30)
+
+/obj/item/device/black_market_scanner/Initialize()
+	. = ..()
+	update_icon()
+
+/obj/item/device/black_market_scanner/update_icon(var/scan_value = 0, var/scanning = FALSE)
+	. = ..()
+	overlays.Cut()
+	overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_flash")
+	if(scanning)
+		overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_clamp_on")
+		switch(scan_value)
+			if(0)
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_red")
+			if(1 to 15)
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_orange")
+			if(15 to 20)
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_yellow")
+			if(25 to 30)
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_green")
+			if(35 to 49)
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_cyan")
+			else
+				overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_value_white")
+		addtimer(CALLBACK(src, PROC_REF(update_icon)), 1 SECONDS)
+	else
+		overlays += image('icons/obj/items/devices.dmi', "+mendoza_scanner_clamp_off")
+
+/obj/item/device/black_market_scanner/afterattack(var/atom/hit_atom, mob/user, proximity)
+	if(!proximity)
+		return
+	var/market_value = get_black_market_value(hit_atom)
+	if(isnull(market_value))
+		return ..()
+	market_value = POSITIVE(market_value)
+	user.visible_message(SPAN_WARNING("[user] presses a button on the [src] and holds it over \the [hit_atom]..."), SPAN_WARNING("You scan \the [hit_atom]..."))
```

Comment:
\the is implict

Reply:
wat?

Reply:
```
/obj/name = "foo"

var/obj/O = new

world.log << "[O]"
world.log << "\the [O]"
```
will both output "the foo", the \the is *implied*